core/arch/arm/include/mm/mobj.h

```diff
@@ -47,6 +47,12 @@ static inline void *mobj_get_va(struct mobj *mobj, size_t offset)
 	return NULL;
 }
 
+/*
+ * Get the physical address for a mobj memory cell
+ * Return TEE_ERROR_NO_DATA if mobj refers to volatile physical memory
```

Comment:
"volatile" is not well defined here, I suggest: "refers to memory that is pageable and pager is enabled"

Reply:
ok